### PR TITLE
Added script libraries rest api doc

### PR DESCRIPTION
**Pull request by Wathsara**
This PR Subjected to - [Script Library REST API](https://github.com/wso2/product-is/issues/7302)

### CODE REVIEW

File: en/docs/develop/scriptLibrary-rest-api.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    ],
+    layout: "StandaloneLayout"
+  })
+  // End Swagger UI call region
+
+  window.ui = ui
+}
+</script>
+
+[![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/88672dbc6afe81e8c5d2)
```
Comment (ayshsandu) [inline en/docs/develop/scriptLibrary-rest-api.md]: Update postman collection

File: en/docs/develop/restapis/scriptLibrary.yaml [hunk 2]
```diff
@@ -0,0 +1,449 @@
...
+        self:
+          type: string
+          example: "/t/wso2.com/api/server/v1/script-libraries/test.js"
+
+    ScriptLibraryResponse:
+      type: object
+      required:
+        - "name"
+        - "content-ref"
+      properties:
```
Comment (ayshsandu) [inline en/docs/develop/restapis/scriptLibrary.yaml]: Explain all the properties

File: en/docs/develop/restapis/scriptLibrary.yaml [hunk 3]
```diff
@@ -0,0 +1,473 @@
...
+              schema:
+                $ref: '#/components/schemas/Error'
+  /script-libraries/{script-library-name}:
+    get:
+      tags:
+        - Script Libraries
+      summary: |
+        Get script library.
+      description: |
+        This API provides the capability to retrive the script library details by using its name.<br>
```
Comment (gomathyk) [inline en/docs/develop/restapis/scriptLibrary.yaml]: to retrieve the details of the script library using its name

File: en/docs/develop/restapis/scriptLibrary.yaml [hunk 4]
```diff
@@ -0,0 +1,473 @@
...
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+    delete:
+      tags:
+        - Script Libraries
+      summary: |
+        Delete a script library.
+      description: |
+        This API provides the capability to delete an script library by  giving its name.<br>
```
Comment (gomathyk) [inline en/docs/develop/restapis/scriptLibrary.yaml]: delete a script library using its name

File: en/docs/develop/restapis/scriptLibrary.yaml [hunk 5]
```diff
@@ -0,0 +1,473 @@
...
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+    put:
+      tags:
+        - Script Libraries
+      summary: |
+        Update a script library.
+      description: |
+        This API provides the capability to Update a script library of an script library by using name.<br>
```
Comment (gomathyk) [inline en/docs/develop/restapis/scriptLibrary.yaml]: to update a script library.

File: en/docs/develop/restapis/scriptLibrary.yaml [hunk 6]
```diff
@@ -0,0 +1,473 @@
...
+        description: This represents the script library details to be updated.
+        required: true
+  /script-libraries/{script-library-name}/content:
+    get:
+      tags:
+        - Script Libraries
+      summary: |
+        Get the content of the script library.
+      description: |
+        This API provides the capability to retrive the content of script library by using its name.<br>
```
Comment (gomathyk) [inline en/docs/develop/restapis/scriptLibrary.yaml]: to retrieve the content of the script library using it name.

File: en/docs/develop/scriptLibrary-rest-api.md [hunk 7]
```diff
@@ -0,0 +1,37 @@
+---
+template: templates/swagger.html
+---
+??? Note "Note: Click to view"
```
Comment (gomathyk) [inline en/docs/develop/scriptLibrary-rest-api.md]: ??? note "Click to view"

File: en/docs/develop/scriptLibrary-rest-api.md [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+    Do the following to try out the REST APIs with your local instance of WSO2 Identity Server.
```
Comment (gomathyk) [inline en/docs/develop/scriptLibrary-rest-api.md]: Follow the steps given below to try out the REST APIs with your local instance of WSO2 Identity Server.

File: en/docs/develop/scriptLibrary-rest-api.md [hunk 9]
```diff
@@ -0,0 +1,37 @@
...
+
+    1.  Expand the relevant API operation and click the **Try It Out** button.
```
Comment (gomathyk) [inline en/docs/develop/scriptLibrary-rest-api.md]: click **Try it Out**.

File: en/mkdocs.yml [hunk 10]
```diff
@@ -541,6 +541,7 @@ nav:
             - 'Secondary User Store': develop/userstore-rest-api.md
             - 'Keystore Management': develop/keystore-rest-api.md
             - 'Identity Providers': develop/idp-rest-api.md
+            - 'Script Libraries': develop/scriptLibrary-rest-api.md
```
Comment (sherenewso2) [inline en/mkdocs.yml]: let's avoid capital letters in any of the file names as a best practice?

**Pull request merged**